### PR TITLE
Bugfix: Vertical Scrollbar flickering during tracking

### PR DESCRIPTION
**Pull request by Maaiins**
Ref #264

### CODE REVIEW

**Comment by Maaiins** [comment]
Commit looks correct

File: modules/base/js/owa.tracker-combined-min.js [hunk 1]
```diff
@@ -125,7 +123,7 @@ if(delimiter===true){delimiter='1';}
 if(!limit){return string.toString().split(delimiter.toString());}else{var splitted=string.toString().split(delimiter.toString());var partA=splitted.splice(0,limit-1);var partB=splitted.join(delimiter.toString());partA.push(partB);return partA;}},isIE:function(){if(/MSIE (\d+\.\d+);/.test(navigator.userAgent)){return true;}},getInternetExplorerVersion:function(){var rv=-1;if(navigator.appName=='Microsoft Internet Explorer')
 {var ua=navigator.userAgent;var re=new RegExp("MSIE ([0-9]{1,}[\.0-9]{0,})");if(re.exec(ua)!=null)
 rv=parseFloat(RegExp.$1);}
-return rv;},isBrowserTrackable:function(){var dntProperties=['doNotTrack','msDoNotTrack'];for(var i=0,l=dntProperties.length;i<l;i++){if(navigator[dntProperties[i]]==='yes'){return false;}}
+return rv;},isBrowserTrackable:function(){var dntProperties=['doNotTrack','msDoNotTrack'];for(var i=0,l=dntProperties.length;i<l;i++){if(navigator[dntProperties[i]]&&navigator[dntProperties[i]]=="1"){return false;}}
```
Comment (padams) [inline modules/base/js/owa.tracker-combined-min.js]: why is this line in the diff for this file but not for the owa.tracker.js it is built off of?

Comment (padams) [reply modules/base/js/owa.tracker-combined-min.js]: I think this change is from a prior commit so something is out of sync with your branch maybe.

Comment (Maaiins) [reply modules/base/js/owa.tracker-combined-min.js]: image

There is the owa.tracker.js. I don't think it´s  out of sync my master is even. Take a look at https://github.com/Maaiins/Open-Web-Analytics/tree/master

Comment (padams) [reply modules/base/js/owa.tracker-combined-min.js]: Right but why then is the diff for owa.tracker-combined-min.js have that extra change?

Comment (Maaiins) [reply modules/base/js/owa.tracker-combined-min.js]: `owa.tracker-combined-min.js` is out of sync in master
See: https://github.com/Open-Web-Analytics/Open-Web-Analytics/blob/a85177e5203c1331388e384e542511593cbe460d/modules/base/js/owa.js#L1912
And: https://github.com/Open-Web-Analytics/Open-Web-Analytics/blob/a85177e5203c1331388e384e542511593cbe460d/modules/base/js/owa.tracker-combined-min.js#L128

**Comment by padams** [comment]
Ack. looks like this commit backed out the change and I didn't catch it. 7cae901bb89a22970bc7fa890cfc07f80b89194a

I'll go ahead and merge your change which should bring everything back up to current.

**Pull request merged**